flutter-candidate.txt

```diff
@@ -1 +1 @@
-3.20.0-16.0.pre
```

Comment:
this file shouldn't be updated manually. Our github actions take care of this automatically to make sure our version is consistent with what is in google3.